internal/page/app_data.go

```diff
@@ -9,26 +9,27 @@ import (
 )
 
 type AppData struct {
-	Page             string
-	Path             string
-	Query            string
-	Localizer        Localizer
-	Lang             localize.Lang
-	CookieConsentSet bool
-	CanGoBack        bool
-	SessionID        string
-	RumConfig        RumConfig
-	StaticHash       string
-	Paths            AppPaths
-	LpaID            string
-	CsrfToken        string
-	ActorTypes       actor.Types
-	ActorType        actor.Type
-	AttorneyID       string
-	OneloginURL      string
-	AppPublicURL     string
-	IsSupporter      bool
-	OrganisationName string
+	Page                 string
+	Path                 string
+	Query                string
+	Localizer            Localizer
+	Lang                 localize.Lang
+	CookieConsentSet     bool
+	CanGoBack            bool
+	SessionID            string
+	RumConfig            RumConfig
+	StaticHash           string
+	Paths                AppPaths
+	LpaID                string
+	CsrfToken            string
+	ActorTypes           actor.Types
+	ActorType            actor.Type
+	AttorneyID           string
+	OneloginURL          string
+	AppPublicURL         string
+	IsSupporter          bool
+	OrganisationName     string
+	IsManageOrganisation bool
```

Comment:
at some point I want to figure out a better way of organisation all this stuff

Reply:
It is getting quite chunky now 🤔